quickstart/csharp/dotnet/text-to-speech/helloworld/Program.cs

```diff
@@ -14,11 +14,22 @@ class Program
     {
         public static async Task SynthesisToSpeakerAsync()
         {
+            // To support Chinese Characters
+            Console.InputEncoding = System.Text.Encoding.Unicode;
+            Console.OutputEncoding = System.Text.Encoding.Unicode;
+
             // Creates an instance of a speech config with specified subscription key and service region.
             // Replace with your own subscription key and service region (e.g., "westus").
             // The default language is "en-us".
             var config = SpeechConfig.FromSubscription("YourSubscriptionKey", "YourServiceRegion");
 
+            // Generally, you should not set "EndpointId" param, but if you are using your own Custom Voice,
```

Comment:
I am not sure if we should put endpoint id in the `helloworld` codes. The codes here should be simple to show how to light up. We have custom voice sample under `samples` folder.

Reply:
> I am not sure if we should put endpoint id in the `helloworld` codes. The codes here should be simple to show how to light up. We have custom voice sample under `samples` folder.

I remove codes about endpointid